.github/workflows/ci.yml

```diff
@@ -190,7 +205,7 @@ jobs:
         with:
           repository: jalantechnologies/github-ci
           path: platform
-          ref: ${{ inputs.ci_version }}
+          ref: ${{ needs.build.outputs.ci_version }}
```

Comment:
Needs to be `needs.build.outputs.workflow_ci_version` (Hint: See how `image_ref ` output is defined in `build`)